.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -375,3 +448,13 @@ jobs:
         grep 'exit code: [^0]' -r logs >> $GITHUB_STEP_SUMMARY
 
         exit $ret
+    - name: Pack e2e binaries
+      if: ${{ always() && !cancelled() && inputs.upload_artifact == 'true'}}
```

Comment:
Why `inputs.upload_artifact`? Can't we use build-only mode as the condition?

Also, how much space is this going to require (mostly interested in comparison with the toolchain artifact itself).

Reply:
The size of the artifact is 270MB right now.

right now build-only/run-only modes are set through the extra_lit_opts input, which may have other options included in it. so I needed another variable to tell me if I should upload the binaries. 
What I could do instead is make a parameter which we would set to build-only/run-only/full and use that in these checks, as well as use that to add the lit option for the testing mode. Open to other ideas as well.

Reply:
i think if we set the retention days to like 1 day or something it shouldnt be that bad?